integration_tests/src/test/scala/com/nvidia/spark/rapids/UnaryOperatorsSuite.scala

```diff
@@ -37,6 +37,6 @@ class UnaryOperatorsSuite extends SparkQueryCompareTestSuite {
   }
 
   testSparkResultsAreEqual("Test murmur3", mixedDfWithNulls) {
-    frame => frame.selectExpr("hash(longs, doubles, 1, null, 'stock string', ints, strings)")
```

Comment:
Do we want an xfail test with a corresponding issue for double hashing?  This looks like we're punting on the issue in a different way.

Reply:
I personally would be fine completely deleting the test.  It did not cover much of anything and what we have now is much better. I will mark the double/float tests as xfail for now and point them to an issue to fix them.